src/components/Sidenav/Sidenav.types.ts

```diff
@@ -0,0 +1,21 @@
+import { ReactNode } from 'react';
+
+import { SidenavClasses } from './Sidenav.classes';
+
+import { SxProps, Theme } from '@mui/material';
+
+export interface SidenavProps {
+  children?: [ReactNode, ReactNode];
+  /** Override or extend the styles applied to the component. */
+  sx?: SxProps<Theme>;
+  /** The system prop that allows defining system overrides as well as additional CSS styles. */
+  classes?: Partial<SidenavClasses>;
+  /** Class applied to the root element. */
+  className?: string;
+  /** If `true`, the component is shown. */
```

Comment:
Тоже не идеально, но лучше.
```suggestion
/** Whether the sidebar should be displayed. */
```